Tests/Functional/Controller/LoginControllerTest.php

```diff
@@ -66,10 +68,10 @@ public function testRedirectingToRegistrationFormWithError(): void
 
     public function testLoginPageWithError(): void
     {
+        self::$container->set(ClientInterface::class, $this->createMock(ClientInterface::class));
+
         $client = static::createClient();
-        $httpClient = $this->prophesize(ClientInterface::class);
-        $client->getContainer()->set(ClientInterface::class, $httpClient->reveal());
-        $session = $client->getContainer()->get('request_stack')->getSession();
+        $session = self::$container->get('session');
```

Comment:
the session as a service was deprecated in Symfony 5.3 afaik

Reply:
True, previous was more correct one, will revert.

Reply:
Restored & refactored a bit to use shortcuts from newest Symfony.